### PR TITLE
Add CHANGELOG-template.rst

### PR DESCRIPTION
**Pull request by lokijuhy**
Added a template for the `CHANGELOG.rst`, so it's clear how the changelog should be formatted, and you don't have to put together a hodgepodge of copy-pasting to assemble each new changelog entry.

Also fixed a small CHANGELOG formatting mishap I noticed, in which `0.24.0` had 2 "Internal Changes" sections.

### CODE REVIEW

File: CHANGELOG-template.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- 🐸 **Team**: changelog item with emoji, ordered by importance to users
+
+**✨ Improvements**
+
+- 🐸 **Team**: changelog item with emoji, alphabetized by team name
+
+**🐞 Bug Fixes**
+
+- 🐸 **Team**: changelog item with emoji, alphabetized by team name
```
Comment (lorenzo-cavazzi) [inline CHANGELOG-template.rst:19]: Bug fixes don't have emojis in the previous releases. Is the emoji intended here (i.e. it's going to change in the next releases) or should it rather be removed also from here

File: CHANGELOG-template.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- **Team**: changelog item, alphabetized by team name
+
+**Bug Fixes**
+
+- **Team**: changelog item, alphabetized by team name
+
+Individual components
+~~~~~~~~~~~~~~~~~~~~~~
+
+- component links, alphabetized by team name
```
Comment (lorenzo-cavazzi) [inline CHANGELOG-template.rst:35]: Should we alphabetize them in the last release too, to have it as an example?
I guess it slipped through but we can easily fix it in this PR

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
Good idea!
I added a couple of notes inline

File: CHANGELOG-template.rst (hunk 1 above)
Comment (lokijuhy) [reply CHANGELOG-template.rst]: Oh yes, thank you, fixed!

File: CHANGELOG-template.rst (hunk 2 above)
Comment (lokijuhy) [reply CHANGELOG-template.rst:35]: You already did this... thank you... 🙇

**Review by bethcg — APPROVED** [review]
Thank you! The template is extremely useful :) !

**Review by lorenzo-cavazzi — APPROVED** [review]

**Review by aledegano — APPROVED** [review]

**Pull request merged**